fabric-engine/package.json

```diff
@@ -5,8 +5,7 @@
   "main": "index.ts",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "asc index.ts -b build/untouched.wasm -t build/untouched.wat --use Math=JSMath --sourceMap --validate",
-    "build:prod": "asc index.ts -b build/fabric.wasm -t build/optimized.wat --sourceMap --validate --optimize",
+    "build": "asc index.ts -b build/fabric.wasm -t build/optimized.wat --sourceMap --validate --optimize",
```

Comment:
No need for "untouched" build anymore?

Reply:
Never really was. I guess that's for really analyzing what WASM gets generated, since optimization makes it hard to read.